util/config/config.go

```diff
@@ -95,38 +96,38 @@ func Init() error {
 		Parameters.IncrementPort()
 	}
 
-	if Parameters.NAT {
+	if Parameters.NAT && !SkipNAT {
```

Comment:
@yilunzhang The config.Parameters.NAT default is false. This condition will never true.